meminfo.c

```diff
@@ -56,17 +57,25 @@ static long long available_guesstimate(const char* buf)
     return MemFree + Cached + Buffers - Shmem;
 }
 
-/* Parse /proc/meminfo.
+/* Parse /proc/meminfo and other related files:
+ *   * ZFS's /proc/spl/kstat/zfs/arcstats, if it exists
  * This function either returns valid data or kills the process
  * with a fatal error.
  */
 meminfo_t parse_meminfo()
 {
     static FILE* fd;
+    static FILE* arcstats_fd;
```

Comment:
Not sure if we should keep this fd always-open. zfs is a kernel module that can be unloaded, right?

Reply:
That's a good point, I've changed it and added a comment.